models/src/main/protobuf/RhoTypes.proto

```diff
@@ -134,7 +134,8 @@ message Receive {
     repeated ReceiveBind binds = 1;
     Par body = 2 [(scalapb.field).no_box = true];
     bool persistent = 3;
-    int32 bindCount = 4;
+    bool peek = 4;
```

Comment:
TODO: replace replace presistent & peek with a single field holding an ADT, e.g. `BindType = Normal | Persistent | Peek`

Reply:
I'm hesitating with introducing this change because it would have to be reverted if we decided to move the peek flag to `ReceiveBind`

Reply:
Ok, in that case I'll skip this change